oauth2_provider/tests/test_token_view.py

```diff
@@ -0,0 +1,128 @@
+                                   expires=timezone.now() + datetime.timedelta(days=1),
+                                   scope='read write')
+
+        response = self.client.get(reverse('oauth2_provider:authorized-token-list'))
+        self.assertEqual(response.status_code, 200)
+        self.assertNotIn(b'There are no authorized tokens yet.', response.content)
+
+    def test_list_view_shows_correct_user_token(self):
+        """
+        Test that only logged-in user's token are shown
+        """
+        self.client.login(username="bar_user", password="123456")
+        AccessToken.objects.create(user=self.foo_user, token='1234567890',
+                                   application=self.application,
+                                   expires=timezone.now() + datetime.timedelta(days=1),
+                                   scope='read write')
+
+        response = self.client.get(reverse('oauth2_provider:authorized-token-list'))
+        self.assertEqual(response.status_code, 200)
+        self.assertIn(b'There are no authorized tokens yet.', response.content)
+
+    def test_delete_view_authorization_required(self):
+        self.token = AccessToken.objects.create(user=self.foo_user, token='1234567890',
+                                                application=self.application,
+                                                expires=timezone.now() + datetime.timedelta(days=1),
+                                                scope='read write')
+
+        response = self.client.get(reverse('oauth2_provider:authorized-token-delete', kwargs={'pk': self.token.pk}))
+        self.assertEqual(response.status_code, 302)
+
+    def test_delete_view_works(self):
+        self.token = AccessToken.objects.create(user=self.foo_user, token='1234567890',
+                                                application=self.application,
+                                                expires=timezone.now() + datetime.timedelta(days=1),
+                                                scope='read write')
+
+        self.client.login(username="foo_user", password="123456")
+        response = self.client.get(reverse('oauth2_provider:authorized-token-delete', kwargs={'pk': self.token.pk}))
+        self.assertEqual(response.status_code, 200)
+
+    def test_delete_view_token_belongs_to_user(self):
+        """
+        Only show if token belongs to logged-in user!
+        """
+        self.token = AccessToken.objects.create(user=self.foo_user, token='1234567890',
+                                                application=self.application,
+                                                expires=timezone.now() + datetime.timedelta(days=1),
+                                                scope='read write')
+
+        self.client.login(username="bar_user", password="123456")
+        response = self.client.get(reverse('oauth2_provider:authorized-token-delete', kwargs={'pk': self.token.pk}))
+        self.assertEqual(response.status_code, 404)
+
```

Comment:
you should also test that a 'POST' cannot delete the token if it does not belong to the user